docs/components/component-lifecycle.md

```diff
@@ -13,8 +13,9 @@ Components have numerous lifecycle methods which can be used to know when the co
 
 Implement one of the following methods within a component class and Stencil will automatically call them in the right order:
 
-<lifecycle-chart></lifecycle-chart>
+import LifecycleMethodsChart from '@site/src/components/LifecycleMethodsChart';
```

Comment:
Probably a dumb question, but how does this import get resolved? Are we defining this path somewhere, or is this some docusaurus magic?

Reply:
I believe it is magic 🪄💫🌟

I followed what it recommends here: https://docusaurus.io/docs/markdown-features/react#importing-components

I think it injects this `@site` scope into the build somehow